torch_xla/experimental/distributed_checkpoint/planners.py

```diff
@@ -36,9 +36,10 @@
 from torch.utils._pytree import tree_map
 from torch_xla.distributed.spmd import XLAShardedTensor, XLAShard
 from torch_xla.experimental.distributed_checkpoint._helpers import (
-    FLATTEN_MAPPING, flatten_state_dict, dedup_tensors, _is_sharded_tensor,
-    set_element, narrow_tensor_by_index, _unwrap_xla_sharded_tensor, _CpuShards)
+    FLATTEN_MAPPING, flatten_state_dict, _is_sharded_tensor, set_element,
```

Comment:
do we need a test?

Reply:
Oh I added one but missed it with git, let me update the PR